experimental/ExtendTeamsforM365/client/pages/myOrders.html

```diff
@@ -5,6 +5,29 @@
     <meta charset="UTF-8" />
     <title>Northwind Orders</title>   
     <link rel="icon" href="data:;base64,="> <!-- Suppress favicon error -->
+    <style>
```

Comment:
move to css file.

Reply:
Changed and shifted to northwind-teams.css